docs/application/native/guides/machine-learning/overview.md

```diff
@@ -13,6 +13,10 @@ You can use the following machine learning features in your native applications:
 
   You can construct, control, and train a machine learning model on-the-fly. The training can be done using the code or with a set of format that describes the model. These models can be trained on the device locally when needed.
 
+- [Machine Learning Service](machine-learning-service.md)
+
+  The Machine Learning Service API provides interfaces to store and fetch the pipeline description for AI application developers.
+
 ## Related Information
```

Comment:
Please use sentence-style capitalization for second headings. Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization 
```suggestion
## Related information
```